signer/openrpc.json

```diff
@@ -145,43 +97,158 @@
     {
       "name": "jade_newAccount",
       "summary": "Creates a new account and stores it locally as a passphrase-encoded keystore file.",
-      "params": [],
+      "params": [
+        {
+          "name": "newAccount",
+          "description": "Creates a new account and stores it locally as a passphrase-encoded keystore file.",
+          "schema": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "account name",
+                "type": "string"
+              },
+              "description": {
+                "description": "account description",
+                "type": "string"
+              },
+              "passphrase": {
+                "$ref": "#/components/schemas/Passphrase"
+              }
+            }
+          }
+        },
+        {
+          "$ref": "#/components/schemas/Additional"
+        }
+      ],
       "result": {
         "name": "newAccountResult",
         "schema": {
-          "type": "boolean"
+          "$ref": "#/components/contentDescriptors/Address"
         }
-      }
+      },
+      "examples": [
+        {
+          "name": "newAccountExample",
+          "description": "Example of creating a new account",
+          "params": [
+            {
+              "name": "testAccount",
+              "value": {
+                "name": "my test wallet",
+                "description": "my test wallet description",
+                "passphrase": "baz"
+              }
+            },
+            {
+              "name": "additionalExample",
+              "value": {
+                "chain": "etc"
+              }
+            }
+          ],
+          "result": {
+            "name": "newAccountResultExample",
+            "value": "0xae6645b12af5f3d76d854f9cbe514476f9b110fb"
+          }
+        }
+      ]
     },
     {
       "name": "jade_signTransaction",
       "summary": "Signs transaction offline with private key from keystore file with given passphrase.\nIf `function` and `arguments` are provided, they will be encoded according smart contract ABI \nand used in the `data` field of the transaction",
-      "params": [],
+      "params": [
+        {
+          "name": "transaction",
+          "description": "transaction to sign",
+          "schema": {
+            "$ref": "#/components/schemas/Transaction"
+          }
+        },
+        {
+          "$ref": "#/components/contentDescriptors/Additional"
+        }
+      ],
       "result": {
-        "name": "signTransactionResult",
-        "description": "",
+        "name": "signedTransaction",
```

Comment:
The above methods result is `newAccountResult`, how come we are changing that pattern for this one?

Reply:
sounded better lol 😂 thoughts?

Reply:
I dont have any pref, just as long as its consistent